src/components/events/commonEventFields/additionalDetails.tsx

```diff
@@ -1,9 +1,9 @@
 import { Grid, TextField, Button } from '@mui/material';
 import { ExpandMore, ChevronLeft } from '@mui/icons-material';
 import { IEvent } from '../../../types/interfaces';
-import { eventKeyDisplay } from '../../../types/configurations';
+import { EVENT_KEY_DISPLAY } from '../../../types/configurations';
 
-interface DetailsProps {
+interface Props {
   details: string[],
```

Comment:
`details: (keyof IEvent)[],` ? and remove all `as keyof IEvent`,

Reply:
Done